basalt.lua

```diff
@@ -0,0 +1,18 @@
+-- https://github.com/Pyroxenium/Basalt/
+
+-- TODO: [Github releases provider](https://github.com/unicornpkg/wing/issues/7)
+-- https://github.com/Pyroxenium/Basalt/releases/download/v1.6.3/basalt.lua
+
+local package = {}
+package.name = "basalt"
+package.version = "1.6.3"
+package.instdat = {}
+package.instdat.repo_owner = "pyroxenium"
+package.instdat.repo_name = "basalt"
```

Comment:
For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_name = "Basalt"
```